src/components/Product.js

```diff
@@ -1,15 +1,31 @@
-import React, { useEffect, useState } from 'react';
-import { getAllProducts } from '../api';
-
+import React from 'react';
+import './Product.css';
 
 const Product = (props) => {
-    const [product, setProduct] = useState([]);
-
-    return <>
-
-    <div>PRODUCTS Component</div>
+    const {id, name, description, price, imageURL, inStock, category} = props.product;
```

Comment:
Create a new `useEffect` hook here, that fetches the single product, based on the productId in the prarams.  
```js
const params = useParams();
const {productId} = params;
```